docker/apps/agreement_database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vickumar/python-nodejs
+
+RUN git clone https://github.cfpb.gov/CFPB/agreement_database.git /apps/agreement_database/
```

Comment:
@chosak   I'm not sure I understand.  Each project does have its own dockerfile.  This is the dockerfile for one project, agreement database.  The docker-compose file links the dockerfiles for each of the apps/projects with the main, cf.gov, container as well as links in the mysql and elasticsearch containers.